batch/plugins/org.jboss.tools.batch.core/META-INF/MANIFEST.MF

```diff
@@ -5,7 +5,9 @@ Bundle-Activator: org.jboss.tools.batch.core.BatchCorePlugin
 Bundle-Vendor: %providerName
 Export-Package: org.jboss.tools.batch.core,
  org.jboss.tools.batch.internal.core,
- org.jboss.tools.batch.internal.core.impl
+ org.jboss.tools.batch.internal.core.impl,
+ org.jboss.tools.batch.internal.core.preferences,
+ org.jboss.tools.batch.internal.core.validation
```

Comment:
All *.internal packages should marked as internal with test plugins listed as friends. For example: org.jboss.tools.batch.internal.core;x-friends:="org.jboss.tools.batch.core.itest"